### PR TITLE
fix: reword Android Things entry

### PR DESCRIPTION
**Pull request by Rexogamer**

### CODE REVIEW

**Comment by unbeatable-101** [comment]
I don't think this is necessary as Android Things is mentioned in the title

**Comment by Rexogamer** [comment]
I'm happy to change the wording but the capitals at the start looked weird
on the site

File: graveyard.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   {
     "dateClose": "2022-01-05",
     "dateOpen": "2015-05-28",
-    "description": "An Android-based embedded operating system (originally named Brillo) aimed to run on Internet of Things (IoT) devices",
+    "description": "Android Things was an Android-based embedded operating system (originally named Brillo) aimed to run on Internet of Things (IoT) devices",
```
Comment (Rexogamer) [inline graveyard.json:5]: This sound better?
```suggestion
    "description": "Things was an Android-based embedded operating system (originally named Brillo) aimed to run on Internet of Things (IoT) devices",
```
also note:
551124F1-B53D-4553-81D8-BAE2326AEFBB

Comment (unbeatable-101) [reply graveyard.json:5]: Oh I see what you mean now, `Android Things` would probably be best then.

**Comment by codyogden** [comment]
Good catch. The name of the product should start the sentence per the spec. If you could, add a period to the end of the sentence and I'll get this merged.

**Comment by Rexogamer** [comment]
> Good catch. The name of the product should start the sentence per the spec. If you could, add a period to the end of the sentence and I'll get this merged.

Done :)

**Pull request merged**